docs/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -34,7 +35,8 @@ component for use within Identity, all that is required is to set two variables:
 | `KEYCLOAK_INIT_<COMPONENT>_ROOT_URL` | The root URL of where the component is hosted | No default    |
 
 :::note
-Identity supports the following values for the `<COMPONENT>` placeholder: `OPERATE`, `OPTIMIZE`, `TASKLIST`, and `WEBMODELER`.
+Identity supports the following values for the `<COMPONENT>` placeholder: `OPERATE`, `OPTIMIZE`, `TASKLIST`,
+and `WEBMODELER`.
```

Comment:
I ran this locally because I wasn't sure why this changed. Still not sure 🤣 but looks fine!

Reply:
Yea I was a little confused by this change too as I don't remember changing it, I assumed it was some form of auto formatting change!